lib/services/api_service.dart

```diff
@@ -36,32 +37,46 @@ class ApiService {
 
   // Send Message using ChatGPT API
   static Future<List<ChatModel>> sendMessageGPT(
```

Comment:
Hello, 
Thanks for the pull request; everything seems good. However, the method here is a bad idea. I understand that OpenAI doesn't provide us with an option to keep track of previously sent messages, but this way is very bad for many reasons.
For example, what if you just need the answer to the last message? What if the number of characters exceeded the capacity? Moreover, the number of tokens will be drained.
Anyway, overall, the work is good, and this code will be added to a different branch.

Reply:
@hadikachmar3 

what if you just need the answer to the last message?

For that purpose, I added an option in the setting bottomsheet to turn off memory. 

What if the number of characters exceeded the capacity? 

For that purpose, In the 2nd commit, it added a check with will disable the memory option when token exceeded